openbas-api/src/main/resources/application.properties

```diff
@@ -123,6 +123,9 @@ minio.bucket=openbas
 minio.port=9000
 minio.access-key=<key>
 minio.access-secret=<secret>
+# S3 configuration
+openbas.s3.use_aws_role=false
+openbas.s3.sts_endpoint=<sts_endpoint>
```

Comment:
**question:**  Wy _ instead of - ? I see that we have - for all the other properties

Reply:
Good point, I'll change it